doc/configuration.md

```diff
@@ -4,6 +4,15 @@ The Ignition configuration is a JSON document conforming to the following specif
 
 * **ignition** (object): metadata about the configuration itself.
   * **version** (string): the semantic version number of the spec. Must be at least `2.0.0`.
+  * **_config_** (objects): options related to the configuration.
+    * **_append_** (list of objects): a list of the configs to be appended to the current config.
+      * **contents** (string): the URL of the config. Supported schemes are http.
```

Comment:
What do y'all think about the name of this key? Elsewhere, we will use `contents.source` for the URL. Maybe this should just be `source`.